payments-core/res/layout/stripe_card_multiline_widget.xml

```diff
@@ -28,12 +28,12 @@
 
         </com.stripe.android.view.CardNumberTextInputLayout>
 
-        <com.stripe.android.view.CardBrandView
+        <com.stripe.android.view.CardBrandViewRebuild
             android:id="@+id/card_brand_view"
             android:layout_width="wrap_content"
             android:layout_height="@dimen/stripe_card_brand_view_height"
             android:layout_marginTop="@dimen/stripe_card_icon_padding"
-            android:layout_marginBottom="@dimen/stripe_card_icon_padding"
+            android:layout_marginBottom="5dp"
```

Comment:
can you make a new variable for this margin, instead of using a constant here?

Reply:
Done 👍